sdk/ml/azure-ai-ml/tests/test_configs/compute/compute-ci-tags.yaml

```diff
@@ -0,0 +1,7 @@
+name: banchci
```

Comment:
do we need a new test config in the SDK just for tags? Would it be possible to instead leverage the `params_override` parameter of the `load_compute` function?

Reply:
I added it out of an abundance of caution, but since the functionality is stable it's not explicitly needed.

Reply:
I've merged this in with `test_compute_instance_load_from_rest`.